graspologic/cluster/autogmm.py

```diff
@@ -117,6 +119,11 @@ class AutoGMMCluster(BaseCluster):
         If provided, min_components and ``max_components`` must match the number of
         unique labels given here.
 
+    n_init : int, optional (default = None)
+        If `n_init` is not `None` and `label_init` is `None` then additional
+        k-means runs will be performed with `n_init` initializations for all covariance
```

Comment:
You may want to update the statement here since now we are adding more runs of GM with 1 initialization.